lesson1/src/stacks_queues/stackProblems.js

```diff
@@ -21,7 +21,15 @@ class StackProblems {
     let stack = new Stack();
     for (let i = 0; i < string.length; i++) {
       let char = string.charAt(i);
-      // todo: do you'r magic ! 🙌
+      let open = {'(': ')' };
```

Comment:
nice! defining an object for this kind of things is certainly not the best practice, but works just as well, the closed object could be replaced by a simple boolean condition like ```char === ')'```